### PR TITLE
24 update readme 1

### PR DESCRIPTION
**Pull request by kang-hyein**

### CODE REVIEW

**Review by nsmader — COMMENTED** [review]
Just a few comments -- a couple about wording, and a couple about a single question I have about whether the new code would work properly.

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ See the `run--00--main-doc.Rmd` script for the most recent description of the st
 	- TECHNICAL: PERNUM, WTFINL, FAMID
 6. Click "View cart"
 7. Click "Create Data Extract"
-8. When the data extract is complete, download the data and the DDI file to the input folder. For the DDI file, right-click on the "DDI" link to save the file in the input folder.
+8. When the data extract is complete, download the data and the DDI file to the input folder. File extension of the data file should appear as `csv.gz` in the input folder. For the DDI file, right-click on the "DDI" link to save the file in the input folder.
```
[the remark this answers is not in the document]
Comment (kang-hyein) [reply README.md:35]: Updated --

File: README.md [hunk 2]
```diff
@@ -194,35 +194,57 @@ kid_age_thres_hs <- 5
...
-# Many auxiliary diagnostics are produced for chunks that only run on the 
-# condition of `eval = developer_mode`. Setting `developer_mode` to FALSE would
-# hide these diagnostics and extra output that should not be present in the final
-# report draft. Set `developer_mode` to TRUE if intending to render these in place 
-# (i.e. throughout the draft) that they are relevant.
-developer_mode <- FALSE
-
 ```
 
+#### Step 4-2. Modify `settings--config.R` by entering information per instruction and save.
```
[the remark this answers is not in the document]
Comment (kang-hyein) [reply README.md:243]: Updated!

File: run--01a--prep-geo-data.Rmd [hunk 3]
```diff
@@ -186,29 +186,29 @@ ggplot() +
...
   }
-  pumaShp <- st_read(glue("{input_path}tl_{tiger_year}_{my_state_fip}_puma20.shp"))
+  pumaShp <- st_read(pumafile)
   rm(pumafile)
 }
 
 if (!exists("zctaShp", envir = .GlobalEnv)) {
   zctafile <- glue("{input_path}tl_{tiger_year}_us_zcta520.shp")
-  if (!exists("zctafile")) {
+  if (!file.exists("zctafile")) {
```
[the remark this answers is not in the document]
Comment (kang-hyein) [reply run--01a--prep-geo-data.Rmd:203]: You were right! Updated

File: run--01a--prep-geo-data.Rmd [hunk 4]
```diff
@@ -400,7 +400,7 @@ if (FALSE) {
 # Crosswalk can be downloaded from the National Center for Education Statistics
 # Reference: https://nces.ed.gov/programs/edge/geographic/relationshipfiles
 grffile <- glue("{input_path}GRF22/grf22_lea_tract.xlsx")
-if (!exists("grffile")) {
+if (!file.exists("grffile")) {
```
[the remark this answers is not in the document]
Comment (kang-hyein) [reply run--01a--prep-geo-data.Rmd]: Done

**Comment by kang-hyein** [comment]
Addressed all comments raised, please approve --

**Review by nsmader — APPROVED** [review]
Review is complete

**Pull request merged**